Code/Source/svFSI/main.cpp

```diff
@@ -242,6 +244,52 @@ void iterate_solution(Simulation* simulation)
     #endif
 
     set_bc::set_bc_dir(com_mod, An, Yn, Dn);
+    
+    if (com_mod.usePrecomp) {
+        #ifdef debug_iterate_solution
+        dmsg << "Use precomputed values ..." << std::endl;
+        #endif
+        // This loop is used to interpolate between known time values of the precomputed
+        // state-variable solution
+        for (int l = 0; l < com_mod.nMsh; l++) {
+            auto lM = com_mod.msh[l];
+            if (lM.Ys.nslices() == 1) {
```

Comment:
Can we avoid the `continue;` statement by only having `if (lM.Ys.nslices() != 1)`?

Reply:
The is not need for  the `} else {` block.

Reply:
removing the `continue` block and simplifying the conditional statement. I will also encapsulate this all in a function now